packages/simplebar/src/simplebar.js

```diff
@@ -389,7 +389,7 @@ export default class SimpleBar {
       'touchend',
       'touchmove'
     ].forEach(e => {
-      this.el.addEventListener(e, this.onPointerEvent, true);
+      this.el.addEventListener(e, this.onPointerEvent, { passive: true });
```

Comment:
Here as you're now passing an object, you removed the `true` argument that was in place previously. That argument was for `capture` event. So I think this should be something like: `{ capture: true, passive: true }`.